src/templates/finops-hub/schema/schema_ea_normalized.json

```diff
@@ -0,0 +1,723 @@
+{
+  "additionalColumns": [
+    {
+      "name": "costInPricingCurrency",
+      "value": "$$COLUMN:CostInBillingCurrency"
+    },
+    {
+      "name": "costInUsd",
+      "value": 0.0
+    },
+    {
+      "name": "exchangeRateDate",
+      "value": "$$COLUMN:Date"
+    },
+    {
+      "name": "exchangeRatePricingToBilling",
+      "value": 1.0
+    },
+    {
+      "name": "invoiceId",
+      "value": ""
+    },
+    {
+      "name": "location",
+      "value": "$$COLUMN:ResourceLocation"
+    },
+    {
+      "name": "paygCostInBillingCurrency",
+      "value": "$$COLUMN:BillingCurrencyCode"
+    },
+    {
+      "name": "paygCostInUsd",
+      "value": 0.0
+    },
+    {
+      "name": "previousInvoiceId",
+      "value": ""
+    },
+    {
+      "name": "pricingCurrency",
+      "value": "$$COLUMN:BillingCurrencyCode"
+    },
+    {
+      "name": "provider",
+      "value": "azure"
+    },
+    {
+      "name": "publisherId",
+      "value": ""
+    },
+    {
+      "name": "resellerMpnId",
+      "value": ""
+    },
+    {
+      "name": "resellerName",
+      "value": ""
+    },
+    {
+      "name": "servicePeriodEndDate",
+      "value": "$$COLUMN:BillingPeriodEndDate"
```

Comment:
Are you sure service period is the entire billing period? Might be better to leave it null/empty.

Reply:
Done